control/cli.py

```diff
@@ -1372,7 +1375,11 @@ def ns_resize(self, args):
         if ns_size <= 0:
             self.cli.parser.error("size value must be positive")
         mib = 1024 * 1024
+        if ns_size % mib:
+            self.cli.parser.error("size value must be aligned to MiBs")
         ns_size = int((ns_size + mib - 1) / mib)    # Convert to MiB
```

Comment:
```suggestion
        ns_size /= mib    # Convert to MiB
```

Reply:
@baum this line stayed from before and I didn't bother to change it. But fine, I'll do that.